crates/bevy_pbr/src/light.rs

```diff
@@ -268,31 +266,131 @@ fn calculate_cascade_bounds(
         .collect()
 }
 
-impl CascadeShadowConfig {
-    /// Returns a cascade config for `num_cascades` cascades, with the first cascade
-    /// having far bound `nearest_bound` and the last cascade having far bound `shadow_maximum_distance`.
-    /// In-between cascades will be exponentially spaced.
-    pub fn new(
-        num_cascades: usize,
-        nearest_bound: f32,
-        shadow_maximum_distance: f32,
-        overlap_proportion: f32,
-    ) -> Self {
+/// Builder for [`CascadeShadowConfig`].
+pub struct CascadeShadowConfigBuilder {
+    num_cascades: usize,
+    minimum_distance: f32,
+    maximum_distance: f32,
+    first_cascade_far_bound: f32,
+    overlap_proportion: f32,
+}
+
+impl CascadeShadowConfigBuilder {
+    /// Constructs a new builder.
+    pub fn new() -> Self {
+        if cfg!(feature = "webgl") {
+            // Currently only support one cascade in webgl.
+            Self {
+                num_cascades: 1,
+                minimum_distance: 0.1,
+                maximum_distance: 100.0,
```

Comment:
Hmm... not sure I agree on that. Having a 1km (assuming world units are metres) single-cascade will look horribly bad. I think it's better to reduce the shadow max distance instead and just not have shadows further away. If people want to, they can change it.

Reply:
This was my intention behind the differing defaults - I felt with only 1 cascade it would be a better tradeoff to just have a shorter shadow distance.